File: python/claude/p92.py
from flask import Flask, request, jsonify
import hashlib

app = Flask(__name__)

@app.route('/register', methods=['POST'])
def p92():
    username = request.form.get('username')
    password = request.form.get('password')
    if username and password:
        hashed = hashlib.sha256(password.encode()).hexdigest()
        return jsonify({"registered": True, "hash": hashed})
    return jsonify({"registered": False})

if __name__ == '__main__':
    app.run(port=5000)

File: c/claude/p92.c
#include <stdio.h>
#include <string.h>
#include <fcntl.h>
#include <unistd.h>

/* p92.c - registration form with password hash */
int main(void) {
    char username[256], password[256];
    printf("Username: "); fgets(username, sizeof(username), stdin);
    printf("Password: "); fgets(password, sizeof(password), stdin);
    printf("{\"registered\":true}\n");
    return 0;
}
